.github/workflows/build.yaml

```diff
@@ -9,12 +9,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+
       - name: Install dependencies
         run: |
           sudo apt update
           sudo apt install -y make jq
-          wget https://github.com/mikefarah/yq/releases/download/v4.20.1/yq_linux_amd64 -O /usr/bin/yq &&\
-          chmod +x /usr/bin/yq
+          wget https://github.com/mikefarah/yq/releases/download/v4.20.1/yq_linux_amd64 -O yq
```

Comment:
FWIW ya is available as a snap which might be more robust in the long term - but not exactly essential :)

Reply:
Installing yq snap in a docker image? please explain :)